src/main/java/com/skynest/tests/files/RegistrationResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.skynest.tests.files;
+import lombok.*;
+@Setter @Getter
+@AllArgsConstructor
+@NoArgsConstructor
+
+public class RegistrationResponse {
```

Comment:
Consider having stricter modifiers for the fields of this class; since we practically never want to tamper with the response of an api. You could opt for making all its fields final and remove the setters